aggregator/src/bin/aggregator.rs

```diff
@@ -185,14 +216,35 @@ pub struct HeaderEntry {
 #[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(untagged)]
 pub enum AggregatorApi {
-    /// Address on which this server should listen for connections to the Janus aggregator API and
-    /// serve its API endpoints, independently from the address on which the DAP API is served.
-    ListenAddress { listen_address: SocketAddr },
-    /// The Janus aggregator API will be served on the same address as the DAP API, but relative to
-    /// the provided prefix. e.g., if `path_prefix` is `aggregator-api`, then the DAP API's uploads
-    /// endpoint would be `{listen-address}/tasks/{task-id}/reports`, while task IDs could be
-    /// obtained from the aggregator API at `{listen-address}/aggregator-api/task_ids`.
-    PathPrefix { path_prefix: String },
+    ListenAddress {
+        /// Address on which this server should listen for connections to the Janus aggregator API and
+        /// serve its API endpoints, independently from the address on which the DAP API is served.
+        listen_address: SocketAddr,
+        /// Resource location at which the DAP service managed by this
+        /// aggregator api can be found on the public
+        /// internet. Required.
+        public_dap_url: Url,
+    },
+    PathPrefix {
+        /// The Janus aggregator API will be served on the same address as the DAP API, but relative to
+        /// the provided prefix. e.g., if `path_prefix` is `aggregator-api`, then the DAP API's uploads
+        /// endpoint would be `{listen-address}/tasks/{task-id}/reports`, while task IDs could be
+        /// obtained from the aggregator API at `{listen-address}/aggregator-api/task_ids`.
+        path_prefix: String,
+        /// Resource location at which the DAP service managed by this
+        /// aggregator api can be found on the public
+        /// internet. Required.
```

Comment:
nit: wrap these comments to the same width as everything else

Reply:
Previously the comments weren't consistently wrapped to 100 so I wasn't sure what to set my editor to. I rewrapped everything to 100 in rebase.